### PR TITLE
[exoBundle] save the change image size  for question graphic

### PR DESCRIPTION
**Pull request by charlotte42**

### CODE REVIEW

**Comment by Elorfin** [comment]
LGTM

File: plugin/exo/Form/InteractionGraphicHandler.php [hunk 1]
```diff
@@ -44,6 +44,14 @@ protected function onSuccessAdd($interGraph)
         $interGraph->getQuestion()->setUser($this->user); // add the user to the question
 
         if ($this->request != null) {
+            $width = $this->request->get('imagewidth'); // Get the width of the image
+            $height = $this->request->get('imageheight'); // Get the height of the image
+
+            $picture = $this->em->getRepository('UJMExoBundle:Picture')->findOneBy(array('id' => $interGraph->getPicture()));
```
Comment (stefk) [inline plugin/exo/Form/InteractionGraphicHandler.php]: @charlotte42, why do you make a database query for `$picture` if you can have it with `$intergrap->getPicture()`?

**Comment by charlotte42** [comment]
I maybe can do it in an other place, I look tomorrow

**Comment by LaurentGruber** [comment]
LGTM

**Pull request merged**